web/modules/custom/dpl_admin/dpl_admin.module

```diff
@@ -228,3 +229,20 @@ function dpl_admin_datetime_set_format(array $element): array {
   $element['time']['#attributes']['step'] = 60;
   return $element;
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ *
+ * The editor can place links in the menu in several levels, but we only
```

Comment:
```suggestion
 * The editor can place links in the main menu in several levels, but we only
```

I think it is worth mentioning that we only have this issue with the main menu. This also links to the first conditional.